src/Utilities.jl

```diff
@@ -190,6 +206,126 @@ function computesymbolsoverrange(
     for (step, θ) in enumerate(rangeiterator)
         θ = collect(θ)
         A = computesymbols(preconditioner, ω, θ)
+        if mass != nothing
+            A = computesymbols(mass, θ)^-1 * A
```

Comment:
Unless there is some magic, this will be a bit cheaper and more stable.
```suggestion
            A = computesymbols(mass, θ) \ A
```